educator/tests/Test/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -348,17 +350,21 @@ spec_StudentApiQueries = describe "Basic database operations" $ do
         it "Last submission is actually the last" $
             sqliteProperty $
               \( delayedGen
-                 (genStudentSignedSubmissions arbitrary (pure submission1))
-                 -> (student, sigSubmissions)
+                 (genCoreTestEnv simpleCoreTestParams) -> env
                ) -> do
-                prepareAndCreateSubmissions sigSubmissions
+                let student = tiOne $ cteStudents env
+                    sigSubs = take 3 $ tiList $ cteSignedSubmissions env
+
+                prepareForSubmissions env
+                forM_ (nub $ toList sigSubs) $ \sigSub -> do
+                    void $ sqlTx $ CoreDB.createSignedSubmission sigSub
+                    liftIO $ threadDelay 10000  -- sqlite keeps time in ms precision
```

Comment:
Eeh, hacky. I guess it's not bad that some txs will have the same timestamp, but we'd like to avoid this for tests only?